File: Nível 9/1047 - Tempo de Jogo com Minutos/main.lua
valores = io.read()
h1, m1, h2, m2 = valores:match("(%S+)%s+(%S+)%s+(%S+)%s+(%S+)")

h1 = tonumber(h1)
h2 = tonumber(h2)
m1 = tonumber(m1)
m2 = tonumber(m2)

minutosInicio = (h1 * 60) + m1
minutosFim = (h2 * 60) + m2

if (minutosFim <= minutosInicio) then
    minutosFim = minutosFim + 24 * 60
end

minutosTotais = minutosFim - minutosInicio

horasTotais = math.floor(minutosTotais / 60)
minutosTotais = minutosTotais - (horasTotais * 60)

print(string.format("O JOGO DUROU %d HORA(S) E %d MINUTO(S)", horasTotais, minutosTotais))

File: Nível 9/1047 - Tempo de Jogo com Minutos/main.py
h1, m1, h2, m2 = map(int, input().split())

minutosInicio = (h1 * 60) + m1
minutosFim = (h2 * 60) + m2

if minutosFim <= minutosInicio:
    minutosFim += 24 * 60

minutosTotais = minutosFim - minutosInicio

horasTotais = int(minutosTotais / 60)
minutosTotais = minutosTotais - (horasTotais * 60)

print(f"O JOGO DUROU {horasTotais} HORA(S) E {minutosTotais} MINUTO(S)")